### PR TITLE
EL-649: Update results page for controlled work outcomes

### PR DESCRIPTION
**Pull request by patrick-laa**
[Jira ticket](https://dsdmoj.atlassian.net/browse/EL-649)

## What changed and why

New copy for results screen for controlled work

## Checklist

Before you ask people to review this PR:

- Tests and rubocop should be passing
- Branch is generally up to date with main Github - definitely no conflicts
- No unnecessary whitespace changes. These make diffs harder to read and conflicts more likely.
- PR description says *what* changed and *why*, with a link to the JIRA story.
- Diff has been checked for unexpected changes being included.
- Commit messages say why the change was made.

### CODE REVIEW

File: spec/system/accessibility_spec.rb [hunk 1]
```diff
@@ -88,9 +88,12 @@
 
   describe "Results page" do
     let(:estimate_id) { SecureRandom.uuid }
+    let(:calculation_result) do
+      CalculationResult.new(build(:api_result)).tap { _1.level_of_help = "certificated" }
+    end
```
Comment (starswan) [inline spec/system/accessibility_spec.rb:93]: Does this (and the change below) mean that we don't have accessibility coverage for the Results page (controlled) or the print results page (certificated) ?

File: spec/features/result_page_spec.rb [hunk 2]
```diff
@@ -3,8 +3,11 @@
 RSpec.describe "Results Page" do
   describe "Client income" do
     let(:estimate_id) { "123" }
+    let(:calculation_result) do
+      CalculationResult.new(payload).tap { _1.level_of_help = "certificated" }
+    end
```
Comment (starswan) [inline spec/features/result_page_spec.rb:8]: have we missed a test for the controlled results page?

File: app/services/cfe_service.rb [hunk 3]
```diff
@@ -25,7 +25,7 @@ def create_assessment_id(cfe_connection, session_data)
 
     def result(cfe_connection, cfe_estimate_id, cfe_session_data)
       cfe_connection.api_result(cfe_estimate_id).tap do |calculation_result|
-        calculation_result.level_of_help = cfe_session_data["level_of_help"]
+        calculation_result.level_of_help = cfe_session_data["level_of_help"] || "certificated"
```
Comment (starswan) [inline app/services/cfe_service.rb:28]: should we avoid doing this if the controlled flag is set? [I did that in another change, and marked it with a 'TODO' when the flag is removed]

File: spec/system/accessibility_spec.rb (hunk 1 above)
Comment (patrick-laa) [reply spec/system/accessibility_spec.rb:93]: Yes. I was being lazy. I will stop being lazy and add the missing tests.

**Review by starswan — APPROVED** [review]

**Pull request merged**